server/package.json

```diff
@@ -7,9 +7,10 @@
     "start": "node app.js",
     "test": "echo \"Error: no test specified\" && exit 0",
     "lint": "prettier-standard './**/*.js'",
-    "dev": "nodemon app.js",
-    "create-db": "echo \"ToDo: create DB\"",
-    "ci-create-db": "echo \"ToDo: create DB\""
+    "dev": "NODE_ENV=development nodemon app.js",
+    "recreate-db:prod": "NODE_ENV=production node ./__db/seed-db.js",
+    "recreate-db": "node ./__db/seed-db.js",
```

Comment:
How about replacing with: 

```
"recreate-db:local": "psql -U postgres -f __db/set-up-local-db.sql && node ./__db/seed-db.js"`
```
and removing `initial-db-setup`, particulary to save confusion about whether than needs to run on heroku or not.